clove/network/bitcoin/contract.py

```diff
@@ -141,6 +182,21 @@ def participate(
         utxo: list=None,
         token_address: str=None,
     ):
+        '''
+        Method for creating a second Atomic Swap transaction based on the secret hash from the current contract.
+
+        Args:
+            symbol (str): network symbol
+            sender_address (str): wallet address of the transaction creator
+            recipient_address (str): wallet address of the transaction recipient
+            value (float): amount to be send
+            utxo (list): list of UTXO objects. In None UTXO will be gathered automatically if needed
+                (for bitcoin-based networks)
+            token_address (str): address of the token contract if we want to use a token
+
+        Returns:
+            BitcoinAtomicSwapTransaction, None: Atomic Swap transaction object or None if something went wrong.o
```

Comment:
It won't necessarily be `BitcoinAtomicSwapTransaction`, it depends on given `symbol`